pandas/tests/frame/methods/test_quantile.py

```diff
@@ -64,66 +81,114 @@ def test_quantile_sparse(self, df, expected):
 
         tm.assert_series_equal(result, expected)
 
-    def test_quantile(self, datetime_frame):
-        from numpy import percentile
-
+    def test_quantile(self, datetime_frame, interp_method):
+        interpolation, method = interp_method
         df = datetime_frame
-        q = df.quantile(0.1, axis=0, numeric_only=True)
-        assert q["A"] == percentile(df["A"], 10)
+        q = df.quantile(
+            0.1, axis=0, numeric_only=True, interpolation=interpolation, method=method
+        )
+        if method == "single":
+            assert q["A"] == np.percentile(df["A"], 10)
         tm.assert_index_equal(q.index, df.columns)
 
-        q = df.quantile(0.9, axis=1, numeric_only=True)
-        assert q["2000-01-17"] == percentile(df.loc["2000-01-17"], 90)
+        q = df.quantile(
+            0.9, axis=1, numeric_only=True, interpolation=interpolation, method=method
+        )
+        if method == "single":
+            assert q["2000-01-17"] == np.percentile(df.loc["2000-01-17"], 90)
```

Comment:
Is this test asserting that the output columns are correct when `method == "table"`?

Reply:
The result will be Series objects, but I improved the assertions here to compare the entire Series results if the interpolation is linear (& method = single) or compare Series name + index if interpolation is nearest (& method = table)